mpas_analysis/ocean/plot_transect_subtask.py

```diff
@@ -399,7 +432,7 @@ def _plot_transect(self, remappedModelClimatology, remappedRefClimatology):
         plt.pcolormesh(x, z, bias)
         plt.colorbar()
         plt.suptitle(title)
-        plt.savefig(outFileName, dpi=200)
+        plt.savefig(outFileName[:-4]+'_OLD.png', dpi=200)
```

Comment:
Please eventually  remove this code, since we don't want to be plotting to old figures.

Reply:
Will do.  I left it in there for now because I've found it useful to compare the plots I am generating with those old ones.  I'll get rid of this code in my next commit, though.